partner_zone/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
[ the # -*- coding: utf-8 -*- line is not necessary anymore in Python 3](https://stackoverflow.com/questions/37048761/coding-utf-8-on-python3/37048998#37048998)